client/styles/pages/home.scss

```diff
@@ -16,7 +24,7 @@
             font-weight: 600;
             font-size: 28px;
             line-height: 1.25;
-            margin: 42px 0 0;
+            margin-top: 0;
```

Comment:
if line 19 is `margin: 0;` then this `margin-top: 0` is unnecessary.

Reply:
ah no, the margin-top is an override!